tox.ini

```diff
@@ -68,6 +72,7 @@ changedir =
   test-core-api: opentelemetry-api/tests
   test-core-sdk: opentelemetry-sdk/tests
   test-core-proto: opentelemetry-proto/tests
+  test-core-auto_instrumentation: opentelemetry-auto-instrumentation/tests
```

Comment:
Not sure why, but when I do `test-core-auto-instrumentation` it doesn't find any tests at all... I had to add an underscore `_` for it to find the tests. Some matching issue I guess?

Reply:
i wouldn't expect the `-` vs `_` to make a difference here

Reply:
Ah I can't explain why it doesn't work with the `-` but I can confirm that it does not :/ It was the difference maker here for some reason.

Even `test-coree-auto-instrumentation` worked just fine. But specifically `test-core-auto-instrumentation` cause `tox` to not find any tests.

How do you want to proceed? Is this a blocker?